src/main/java/ch/njol/skript/effects/EffDetonate.java

```diff
@@ -0,0 +1,80 @@
+package ch.njol.skript.effects;
+
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.World;
+import org.bukkit.block.Block;
+import org.bukkit.block.data.type.Fire;
+import org.bukkit.block.data.type.TNT;
+import org.bukkit.entity.*;
+import org.bukkit.entity.minecart.ExplosiveMinecart;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+
+@Name("Detonate Entity")
+@Description("Immediately detonates an entity or block. Accepted entities are fireworks, TNT minecarts, wind charges, creepers, and tnt. Accepted blocks are tnt.")
```

Comment:
> [...] and tnt. Accepted blocks are tnt.

why do you mention TNT twice? also I think you should keep it consistent, you say TNT minecarts but then you say "tnt" all lowercase, pick one or the other

Reply:
I mention TNT twice because you can spawn it and detonate a places tnt block. For the spelling mistakes, I'll fix that when I can

Reply:
> I mention TNT twice because you can spawn it and detonate a places tnt block. For the spelling mistakes, I'll fix that when I can

I don't think thats very obvious to the user, they might get a bit confused when reading the description. Maybe you can say something like `[...] and primed TNT.  Accepted blocks are placed TNT.` or something similar.